### DISCUSSION TITLE
是否加入其他语言和下载源

### DISCUSSION THREAD

**Discussion by kongxia114** [Meloong-Git/PCL#1649] [提问] [answered]
更多的语言：
英语
日文
俄语
繁体中文
简体中文
。。。

**Comment by kongxia114** [comment]
下载源有：
官方的 
MCBBS
BMCLAPI

**Comment by WForst-Breeze** [comment, marked as answer]
下载源中镜像源已包含 MCBBS&BMCLAPI 源，官方源也已支持。（#1570）

多语言功能不会制作，考虑到 UI 用户界面设计问题和下载源国际适配等诸多问题，已被拒绝（#119）
